src/state/actions/dispatchANTInteraction.ts

```diff
@@ -17,71 +16,107 @@ export default async function dispatchANTInteraction({
   workflowName: ANT_INTERACTION_TYPES;
   signer: ContractSigner;
   owner: string;
-  processId: ArweaveTransactionID;
+  processId: string;
   dispatch: Dispatch<TransactionAction>;
 }): Promise<ContractInteraction> {
   let result: AoMessageResult | undefined = undefined;
 
   const antProcess = ANT.init({
-    processId: processId.toString(),
+    processId: processId,
     signer,
   });
 
   try {
     if (!antProcess) throw new Error('ANT provider is not defined');
-    dispatch({
-      type: 'setSigning',
-      payload: true,
-    });
     switch (workflowName) {
       case ANT_INTERACTION_TYPES.SET_NAME:
+        dispatch({
+          type: 'setSigningMessage',
```

Comment:
consider adding a const for `'setSigningMessage'` and reuse on this switch case block

Reply:
I created a wrapper function for use here instead of a constant for the name.